lib/rules/no-multi-spaces.js

```diff
@@ -83,6 +83,34 @@ module.exports = {
             return false;
         }
 
+        /**
+         * Formats value of given comment token for error message by truncating its length.
+         * @param {ASTNode} token comment token
```

Comment:
Oh, you're totally right - I think I usually use `Token` as well and just goofed on that one.